x-pack/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -662,6 +662,38 @@ describe('Lens App', () => {
         });
       });
 
+      it('Shows Save button in create by value mode with no originating app', async () => {
```

Comment:
Is this covering a real use case? When creating new from a dashboard, there is always an originating app, and when creating new from the Visualize Library there isn't any value input.

Reply:
ah! You're totally right!